mtp_cashbook/apps/cashbook/forms.py

```diff
@@ -40,7 +40,7 @@ def clean_transactions(self):
         transactions = self.cleaned_data.get('transactions', [])
         discard = self.data.get('discard') == '1'
         if not transactions and not discard:
-            raise forms.ValidationError(ugettext('Only click ‘Done’ when you’ve selected credits'))
+            self.add_error(None, ugettext('Only click ‘Done’ when you’ve selected credits'))
```

Comment:
set a non-field-specific error message because a linkable form field does not really exist on the page